pkg/option/flags.go

```diff
@@ -245,6 +247,8 @@ func ReadAndSetFlags() error {
 	Config.EventCacheNumRetries = viper.GetInt(KeyEventCacheRetries)
 	Config.EventCacheRetryDelay = viper.GetInt(KeyEventCacheRetryDelay)
 
+	Config.ProcessCacheStaleInterval = viper.GetUint(KeyProcessCacheStaleInterval)
```

Comment:
Could you use `GetDuration`, that takes care of all the seconds minutes whatever https://pkg.go.dev/github.com/spf13/viper#GetDuration?

Reply:
TBH I was just following how intervals were done for other switches, but yes that is a good idea!